docs/phases/11-phase-cache.md

```
@@ -0,0 +1,464 @@
+        return dayBeforeAdvice
+    }
+    
+    return nil
+}
+```
+
+### フォールバック表示時のUI
+
+前日のアドバイスを表示する際:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)